soccer/gameplay/tactics/find_pass_point.py

```diff
@@ -0,0 +1,24 @@
+import robocup
+import constants
+from scipy.optimize import minimize
+
+## Function that optimizes cost function for passing
+# uses scipy minimize using the BFGS method
+def findPassPoint(heuristic) -> robocup.Point:
+    width = constants.Field.Width
+    length = constants.Field.Length
+    initial_guess = [0.00, 0.00]
+    result = minimize(heuristic, initial_guess, bounds=((-width/2.00, width/2.00), (0.00, length)), tol=1e-3)
```

Comment:
I think we should set the max num iterations arg. I'm not sure whether it should just be hard coded or passed in as a param though. The "correct" value can change significantly depending on the speed of execution of the heuristic.